lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -31,6 +31,24 @@
   white-space: nowrap;
 }
 
+// Should be called on a custom class attached to the <body> tag
+@mixin font-size($custom-font-size) {
+  &.nx-body,
```

Comment:
Is it possible to `@include` at the top level (not nested inside a block)? If so, this `&` could be removed.

Reply:
> Is it possible to `@include` at the top level (not nested inside a block)? If so, this `&` could be removed.

It's not, it has to be associated with an element. I've gone further and decided that it would be best to be proscriptive and state that it should be used specifically on `<body>`